File: examples/fizzbuzz.py
"""
FizzBuzz Example

Prints numbers from 1 to 100 with the following substitutions:
- "Fizz" for multiples of 3
- "Buzz" for multiples of 5
- "FizzBuzz" for multiples of both 3 and 5
"""

def fizzbuzz(n=100):
    for i in range(1, n + 1):
        output = ''
        if i % 3 == 0:
            output += 'Fizz'
        if i % 5 == 0:
            output += 'Buzz'
        print(output or i)

if __name__ == '__main__':
    fizzbuzz()

File: examples/fizzbuzz.js
// FizzBuzz example in JavaScript
// Prints numbers from 1 to 100
// For multiples of 3, prints "Fizz"
// For multiples of 5, prints "Buzz"
// For multiples of both 3 and 5, prints "FizzBuzz"

for (let i = 1; i <= 100; i++) {
  let output = '';
  if (i % 3 === 0) output += 'Fizz';
  if (i % 5 === 0) output += 'Buzz';
  console.log(output || i);
}
